src/handler-builder.ts

```diff
@@ -35,6 +35,13 @@ import * as remoteConfig from './providers/remoteConfig';
 import * as storage from './providers/storage';
 import * as testLab from './providers/testLab';
 
+/**
+ * The `HandlerBuilder` class facilitates the writing of functions by producers
+ * of Firebase Extensions (currenly in limited beta).
```

Comment:
Revise to:
The `HandlerBuilder` class facilitates the writing of functions by producers of Firebase Extensions and developers that wish to use gcloud CLI or Cloud Console to deploy their functions. 

(I would actually remove the part of limited beta, it is not true, it's actually in EAP, and we also will definitely forget to update this comment later)